### DISCUSSION TITLE
Technologies Detection

### DISCUSSION THREAD

**Discussion by bizibabe** [projectdiscovery/httpx#1596] [Q&A] [answered]
Hello,

Upon using the "-td" option, I have observed significantly fewer results compared to when utilizing the Wappalyzer web extension. 
I am curious to understand the reason behind this variance in results. 
Could you please provide some insights or clarification on why such a difference exists?

![image](https://github.com/projectdiscovery/httpx/assets/60073248/db925113-df1d-4815-9c67-dfbe4fa7fcb9)
![image](https://github.com/projectdiscovery/httpx/assets/60073248/7c0be6f9-961b-4899-b459-608a16f01522)

Sincerely,
Bizi

**Comment by Ice3man543** [comment, marked as answer]
Hi @bizibabe,

This is happening due to the fact that Wappalyzer chrome extension uses the rendered DOM of the page (which in case of modern frameworks is almost always rendered by javascript). It checks the DOM for properties or checks which allow it to give more results.

Compared to this, httpx just runs the headers, body etc fingerprints on the http response body from the network port.

Hope that clarifies it 👍

**Reply by bizibabe** [reply]
Hi @Ice3man543,

That's clear, thank you for the explanation! 👍

Do you think it would be possible in the future to have detection in headless mode to specifically retrieve DOM information?
